dacScanV3.py

```diff
@@ -96,8 +98,8 @@ def scanAllLinks(args, calTree, vfatBoard):
 
     #try:
     if args.debug:
-        print("| link | vfatN | vfatID | nameX | dacValX | dacValX_Err | nameY | dacValY | dacValY_Err |")
-        print("| :--: | :---: | :----: | :-----: | :-----: | :---------: | :--: | :-----: | :---------: |")
+        print("| link | vfatN | vfatID | dacSelect | nameX | dacValX | dacValX_Err | nameY | dacValY | dacValY_Err |")
+        print("| :--: | :---: | :----: | :--: |:-----: | :-----: | :---------: | :--: | :-----: | :---------: |")
```

Comment:
Slight readability comment, make the bottom row line up (e.g. how it was before).

Reply:
done